Embedded System L1/AVR ATMEGA32/DIO/01-MCAL/01-DIO/Types.h

```diff
@@ -0,0 +1,23 @@
+/******************************************************************************************************
+ * File: Types.h 
+ * Author: Khalid Mosaad Zakaria
+ * Data: 2 july 2021
+ * Description: This File include Typedef for standard data types.
+*******************************************************************************************************/
+
+#ifndef TYPES_H_
+#define TYPES_H_
+
+typedef unsigned char uint8_t;
+typedef unsigned int  uint32_t;
+
+typedef signed char sint8_t;
+typedef signed int  sint32_t;
+
+
+
+
+
+
+#endif
+
```

Comment:
-comments doesn't follow Sprints_CodeGuideLine of headers

![2](https://user-images.githubusercontent.com/44152298/127502084-967ab9b6-4fba-4be7-ba83-0af348f54c31.PNG)

[Sprints_CodeGuideLine.pdf](https://github.com/AhmedEssamMo/Sprints_Wave_9/files/6900596/Sprints_CodeGuideLine.pdf)